gossip/src/crds_value.rs

```diff
@@ -362,6 +362,16 @@ mod test {
 
     #[test]
     fn test_serialize_round_trip() {
+        // Unfortunately doing `Keypair::new_from_array(rng.gen())` gives
```

Comment:
is it maybe preferable to let the code run and save the new hash values where they are checked? This lovely helper is not useful long-term.